components/treelist/accessibility/wai-aria-support.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor TreeList Documentation | TreeList  Accessibility
+description: "Get started with the Telerik UI for Blazor TreeList and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.1."
+tags: telerik,blazor,accessibility,wai-aria,wcag
+slug: treelist-wai-aria-support 
+position: 50 
+---
+
+# Blazor TreeList Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
+Out of the box, the Telerik UI for Blazor TreeList provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The TreeList is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2  AA](https://www.w3.org/TR/WCAG22/) standards](https://www.w3.org/TR/WCAG22/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
+
+## WAI-ARIA
+
+
+This section lists the selectors, attributes, and behavior patterns supported by the component and its composite elements, if any.
+
+
+The TreeList is a composite component that consists of 2 logically separated structural elements:
```

Comment:
Our style guide calls for spelling out single-digit numbers. See [here](https://docs.telerik.com/style-guide/numbers). However, if fixing this requires too much work in the template, you can ignore this suggestion.
```suggestion
The TreeList is a composite component that consists of two logically separated structural elements:
```